dev/com.ibm.ws.security.mp.jwt/resources/com/ibm/ws/security/mp/jwt/resources/MicroProfileJwtMessages.nlsprops

```diff
@@ -134,7 +134,7 @@ JWT_NOT_FOUND_IN_REQUEST.explanation=To perform authentication, a MicroProfile J
 JWT_NOT_FOUND_IN_REQUEST.useraction=Ensure that a MicroProfile JWT is included in the Authorization header or as a POST parameter of the request.
 
 # 0=Exception message
-ERROR_CREATING_JWT_USING_TOKEN_IN_REQ=CWWKS5523E: The MicroProfile JWT feature cannot authenticate the request because a valid JWT cannot be created from the token included in the request. {0}
+ERROR_CREATING_JWT_USING_TOKEN_IN_REQ=CWWKS5523E: The MicroProfile JWT feature cannot authenticate the request because the token included in the request cannot be validated. {0}
```

Comment:
suggested edit:
The MicroProfile JWT feature cannot authenticate the request because the token that is included in the request cannot be validated.

Reply:
done